### DISCUSSION TITLE
Collect showcase

### DISCUSSION THREAD

**Discussion by deliciouslytyped** [xesscorp/skidl#208] [General]
As someone unfamiliar with this tooling, it would be nice if there was a collection of articles, repositories, or otherwise showcasing things people have built with it.

**Comment by devbisme** [comment]
That's a good idea and I'll have to take some time to assemble something like that.

As an interim solution, I built a project last year called RepoRecon. It scans Github for projects that match a set of keywords and places them into a table. [Here](https://devbisme.github.io/RepoRecon/?topic=skidl&filter=&sort=pushed:desc) is the table for SKiDL projects. (There's a good bit of trash in the table from repos that contain the string "skidl" but don't actually have anything to do with SKiDL.)
